decentraland/unity-explorer
# Feat: start-up/teleport recovery on networking issues

## What does this PR change?

The issue is described here https://github.com/decentraland/unity-explorer/issues/2821 in details

## How to test the changes?

1. Launch the explorer
2. Go to the step with the loading screen
3. Simulate an unstable internet connection (Enable/Disable it during the screen)
4. Ensure that you cannot reach a broken state in the app
5. Ensure that all features work fine as if there was no issues with the internet connection

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
